modules/ref_fb_module/src/scaling_fb_impl.cpp

```diff
@@ -239,6 +255,36 @@ void ScalingFbImpl::createSignals()
     outputSignal.setDomainSignal(outputDomainSignal);
 }
 
+void ScalingFbImpl::initStatuses()
+{
+    auto inputStatusType = EnumerationType("InputStatusType", List<IString>(InputDisconnected, InputConnected, InputInvalid));
+    this->context.getTypeManager().addType(inputStatusType);
```

Comment:
This should probably check for whether or not it exists already.

Reply:
Fixed